src/tileLayer.js

```diff
@@ -728,8 +728,8 @@ module.exports = (function () {
       container.append(tile.image);
       container.css({
         position: 'absolute',
-        left: (bounds.left - parseInt(div.attr('offsetx') || 0, 10)) + 'px',
-        top: (bounds.top - parseInt(div.attr('offsety') || 0, 10)) + 'px'
+        left: (bounds.left - parseFloat(div.attr('offsetx') || 0)) + 'px',
```

Comment:
In thinking about this, perhaps I should have forced the svg layer to use `int`s rather than let the html layer use `float`s.  Fundamentally, I was doing nothing to round the svg layer's transform to integers, and the test code that compares different tile renderings to see if they line up was failing.

I was considering this last night, and I should change this back to ints and make the svg layer match -- otherwise the html renderer shows small gaps between tiles when we don't show lower tile levels.